plugins/module_utils/podman/podman_container_lib.py

```diff
@@ -473,6 +474,16 @@ def addparam_log_opt(self, c):
                                           errors='surrogate_or_strict')])]
         return c
 
+    def addparam_log_opt_file_size(self, c):
```

Comment:
I don't  think it will work with a separate function, all log options we add in a function above - `addparam_log_opt`.
You can follow same logic with `max_size` and add it like this for example:

```python
    def addparam_log_opt(self, c):
        for k, v in self.params['log_opt'].items():
            if v is not None:
                if k in ('max_file', 'max_size'):
                    opt = k.replace('_', '-')
                c += ['--log-opt',
                      b"=".join([to_bytes(opt, errors='surrogate_or_strict'),
                                 to_bytes(v,
                                          errors='surrogate_or_strict')])]
        return c
```